src/Native/Runtime/gcenv.h

```diff
@@ -40,6 +40,14 @@
     #define FireEtwPinPlugAtGCTime(PlugStart, PlugEnd, GapBeforeSize, ClrInstanceID) 0
     #define FireEtwGCTriggered(Reason, ClrInstanceID) 0
 
+    #ifndef _INC_WINDOWS
```

Comment:
Why do we need these typedefs in this change? I don't see their usage being added.